### DISCUSSION TITLE
User-Creatable Memes Plugin or App

### DISCUSSION THREAD

**Discussion by gigawhitlocks** [mattermost/community#18] [Fun Ideas]
The existing memes plugin is fun because it allows users to create new image macros to upload to chat, however, the nature of memes and community building is such that the pre-packaged images become stale as the joke that underlies the meme is played out.

For community-building, it would be fun to see a plugin like the memes plugin today, but which allows users to add new memes entirely to the available library, so that communities can fill the plugin with inside jokes and culture to make their server theirs.

**Comment by BenLloydPearson** [comment]
I love this idea. Now I'm imagining running a meme-building contest on our chat server during developer events.
